int64 flg
---
int64[] result
